tests/sources/file.py

```diff
@@ -86,10 +87,15 @@ def on_moved(self, event):
         observer.stop()
 
 
+async def main(queue, args):
+    loop = asyncio.get_event_loop()
+    await loop.run_in_executor(None, sync_main, loop, queue, args)
```

Comment:
After ctrl-c to break the cli often I get `ERROR:asyncio:Task was destroyed but it is pending!`

I did not see same error if I convert it to the same use as https://github.com/ansible/event-driven-ansible/blob/main/plugins/event_source/file_watch.py#L94

One more question - is https://github.com/ansible/event-driven-ansible/blob/main/plugins/event_source/file_watch.py#L88 needed and why?